### PR TITLE
Definition File Editor supports sending mail

### LINKED ISSUE #307: Definition File Editor is missing send-mail keyed value.
As shown in this subject.

### PR DESCRIPTION
**Pull request by msyk**
Close #307

### CODE REVIEW

File: INTER-Mediator-Support/defedit.php [hunk 1]
```diff
@@ -144,6 +153,6 @@
 /*
  * Don't remove comment slashes below on any 'release.'
  */
-//IM_Entry($defContexts, null, array('db-class' => 'DefEditor'), false);
+IM_Entry($defContexts, null, array('db-class' => 'DefEditor'), 2);
```
Comment (matsuo) [inline INTER-Mediator-Support/defedit.php]: Please comment out this line and disable the debug mode.

**Comment by msyk** [comment]
You're right.

File: INTER-Mediator-Support/defedit.php (hunk 1 above)
Comment (msyk) [inline INTER-Mediator-Support/defedit.php]: You're right.

**Comment by matsuo** [comment]
Thanks.

**Pull request merged**